code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -512,6 +511,50 @@
 	M.adjust_bodytemperature(-8 * TEMPERATURE_DAMAGE_COEFFICIENT, BODYTEMP_NORMAL)
 	..()
 
+/datum/reagent/consumable/buzz_fuzz
+	name = "Buzz Fuzz"
+	description = "~A Hive of Flavour!~ NOTICE: Addicting."
+	id = "buzz_fuzz"
+	addiction_threshold = 14
+	color = "#8CFF00" // rgb: 135, 255, 0
+	taste_description = "carbonated honey and pollen"
+	glass_icon_state = "buzz_fuzz"
+	glass_name = "honeycomb of Buzz Fuzz"
+	glass_desc = "Stinging with flavour."
+
+/datum/reagent/consumable/buzz_fuzz/on_mob_life(mob/living/carbon/M)
+	M.reagents.add_reagent("sugar",2)
+	if(prob(25))
+		M.reagents.add_reagent("honey",1)
+
+/datum/reagent/consumable/buzz_fuzz/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(iscarbon(M) && (method in list(TOUCH, VAPOR, PATCH)))
+		var/mob/living/carbon/C = M
+		for(var/s in C.surgeries)
+			var/datum/surgery/S = s
+			S.success_multiplier = max(0.4, S.success_multiplier) // +40% success probability on each step, compared to bacchus' blessing's ~46%
```

Comment:
That's too much, considering this reagent is found in easily accessible roundstart vendors. It kinda overshadows sterilizine.

Reply:
*points to ethonal being better and round start.*

Reply:
To actually surpass sterilizine's surgery success multiplier of 0.2 (20%), the ethanol reagent would require a potency of more than 130 (booze success multiplier being boozepwr / 65 * 0.1), which supposedly only happens for bacchus blessing.